support/SubmitScripts/Wheeler.sh

```diff
@@ -32,20 +32,36 @@ export RUN_DIR=/panfs/ds08/sxs/run/dir
 export SPECTRE_EXECUTABLE=EvolveScalarWave3D
 export SPECTRE_INPUT_FILE=./Input3DPeriodic.yaml
 
-echo "Running on the following nodes:"
-echo ${SLURM_NODELIST}
+export PATH=${SPECTRE_BUILD_DIR}/bin:$PATH
 
 CHARM_PPN=$(expr ${SLURM_CPUS_PER_TASK} - 1)
-echo "Slurm tasks: ${SLURM_NTASKS}"
-echo "Slurm cpus per task: ${SLURM_CPUS_PER_TASK}"
-echo "Charm ppn: ${CHARM_PPN}"
+
+echo "###################################"
+echo "######       JOB INFO        ######"
+echo "###################################"
+echo
+echo "Job ID: ${SLURM_JOB_ID}"
+echo "Run Directory: ${RUN_DIR}"
+echo "Submit Dirrectory: ${SLURM_SUBMIT_DIR}"
+echo "Queue: ${SLURM_JOB_PARTITION}"
+echo "Nodelist: ${SLURM_JOB_NODELIST}"
+echo "Tasks: ${SLURM_NTASKS}"
+echo "CPUs per Task: ${SLURM_CPUS_PER_TASK}"
+echo "PATH: ${PATH}"
```

Comment:
Can you also print `CHARM_PPN` please? This is useful for making sure how Charm++ is launched matches what we expect :)